### PR TITLE
refmt: init at 1.16.0

### PR DESCRIPTION
**Pull request by deemp**
###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.11 Release Notes (or backporting 23.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2305-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Review by emilylange — CHANGES_REQUESTED** [review]
Please edit your PR to target the `master` branch instead of `nixpkgs-unstable`.
Thank you :)

Please also read the [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/da072c0d594456cd309f79785b4a97408e3562ac/CONTRIBUTING.md) (that tick box in your PR description). Specifically "Submitting changes" (as your commit message needs to be `refmt: init at 1.16.0`) and the `meta.description` section.

**Comment by deemp** [comment]
@emilylange, thank you!

**Review by emilylange — APPROVED** [review]

**Review by jsoo1 — APPROVED** [review]
LGTM

**Review by emilylange — CHANGES_REQUESTED** [review]
Oh wait.
Can you please split the change to `maintainers/maintainer-list.nix` into its own separate commit called `maintainers: add deemp`?

So two commits total:
1. `maintainers: add deemp`
1. `refmt: init at 1.16.0`

See https://nixos.org/manual/nixpkgs/unstable/#var-meta-maintainers

**Comment by deemp** [comment]
@emilylange, done!

File: pkgs/development/tools/refmt/default.nix [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    rev = "v${version}";
+    sha256 = "sha256-HiAWSR2S+3OcIgwdQ0ltW37lcG+OHkDRDUF07rfNcJY=";
+  };
+
+  vendorSha256 = "sha256-MiYUDEF9W0VAiOX6uE8doXtGAekIrA1cfA8A2a7xd2I=";
+
+  meta = with lib; {
+    description = "Reformat HCL <-> JSON <-> YAML";
+    homepage = "https://github.com/rjeczalik/refmt";
+    license = licenses.agpl3;
```
Comment (emilylange) [inline pkgs/development/tools/refmt/default.nix:19]: One more thing, sorry^^

```suggestion
    license = licenses.agpl3Only;
```

**Review by emilylange — CHANGES_REQUESTED** [review]

File: pkgs/development/tools/refmt/default.nix (hunk 1 above)
Comment (deemp) [reply pkgs/development/tools/refmt/default.nix]: Done!

**Review by emilylange — APPROVED** [review]
Thank you! :)

**Pull request merged**